src/runtime/runtime.js

```diff
@@ -459,10 +459,15 @@
       return argument;
     }
 
+    var hasNativeSymbol = false;
```

Comment:
I would leave this `undefined` to reinforce that we will set it in one of the `if` branches that will execute before we exit.